### PR TITLE
RIA-732 no fixed address label

### PR DESCRIPTION
**Pull request by fantasticjamieburns**

### CODE REVIEW

File: e2e/features/address-validation.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    Given I am signed in as a Legal Rep
+    And I create a new case
+    And I complete the Home Office reference page
+    And I complete the Basic details page
+
+  @create-case @alternate @RIA-732
+  Scenario: Legal rep indicates there is no fixed address for the appellant
+
+    When I select Yes for the Does the appellant have a fixed address? field
+    When I should not see the text We will use the address of your legal practice.
```
Comment (Rob-Doherty) [inline e2e/features/address-validation.feature]: Think this should be a `Then` for clarity

File: e2e/features/address-validation.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    
+    When I select No for the Does the appellant have a fixed address? field
+    When I should see the text We will use the address of your legal practice.
```
Comment (Rob-Doherty) [inline e2e/features/address-validation.feature]: And a `Then` here as well

Comment (fantasticjamieburns) [reply e2e/features/address-validation.feature]: Quite right

File: e2e/features/address-validation.feature (hunk 1 above)
Comment (fantasticjamieburns) [reply e2e/features/address-validation.feature]: Quite right

**Pull request merged**